ibis/expr/datatypes/core.py

```diff
@@ -96,6 +97,7 @@ class DataType(Annotable, Comparable):
 
     nullable = optional(instance_of(bool), default=True)
 
+    # TODO(kszucs): remove it
```

Comment:
This is useful when parsing, so we do need to some alternative (perhaps a `.copy` if that doesn't already exist?)

Reply:
Yes, we have copy as an alternative available for all Annotable subclasses. I have a follow-up patch which removes it, going to keep the todo note until that.